<?xml version="1.0"?>
<launch>
    <!--Down sampling-->
    <param name="camera/ds_size" type="double" value="0.03" />
    <node pkg="sensor_fusion" type="camera_downsample" name="camera_downsample">
        <remap from="/cloud" to="/zed/point_cloud/cloud_registered" />
        <remap from="/output" to="/zed/ds_cloud" />
    </node>
</launch>
